packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -200,3 +201,62 @@ export const postGetKarma = (post: PostsBase|DbPost): number => {
 export const postCanEditHideCommentKarma = (user: UsersCurrent|DbUser|null, post?: PostsBase|DbPost|null): boolean => {
   return !!(user?.showHideKarmaOption && (!post || !postGetCommentCount(post)))
 }
+
+/**
+ * Returns the event datetimes in a user-friendly format,
+ * ex: Mon, Jan 3 at 4:30 - 5:30 PM
+ * 
+ * @param {(PostsBase|DbPost)} post - The event to be checked.
+ * @returns {string} The formatted event datetimes.
+ */
+export const prettyEventDateTimes = (post: PostsBase|DbPost): string => {
```

Comment:
Yes this is a bit extra. I played around with different datetime formats before landing on something close to how [Facebook events](https://fb.me/e/1jDLjOoyG) handle them.